src/v2/Apps/Artists/artistsRoutes.tsx

```diff
@@ -0,0 +1,74 @@
+import loadable from "@loadable/component"
+import { graphql } from "react-relay"
+import { RouteConfig } from "found"
+
+const ArtistsApp = loadable(() => import("./ArtistsApp"), {
+  resolveComponent: component => component.ArtistsApp,
+})
+
+const ArtistsIndexRoute = loadable(() => import("./Routes/ArtistsIndex"), {
+  resolveComponent: component => component.ArtistsIndexFragmentContainer,
+})
+
+const ArtistsByLetterRoute = loadable(
+  () => import("./Routes/ArtistsByLetter"),
+  {
+    resolveComponent: component => component.ArtistsByLetterFragmentContainer,
+  }
+)
+
+export const artistsRoutes: RouteConfig[] = [
+  {
+    children: [
```

Comment:
For better or worse i've updated all of our routes files with  
```
/* eslint-disable sort-keys-fix/sort-keys-fix */
```
since they follow a pretty linear format -- path, children. We should probably do that here so its not confusing for the next dev